code/_onclick/hud/screen_objects.dm

```diff
@@ -57,11 +57,23 @@
 		M.swap_hand()
 	return 1
 
+/obj/screen/lay
+	name = "lay down"
+	icon = 'icons/mob/screen_midnight.dmi'
+	icon_state = "lay"
+	screen_loc = ui_crafting
+
+/obj/screen/lay/Click()
+	var/mob/living/M = usr
+	if(isobserver(usr))
+		return
+	M.lay_down()
+
 /obj/screen/craft
 	name = "crafting menu"
 	icon = 'icons/mob/screen_midnight.dmi'
 	icon_state = "craft"
-	screen_loc = ui_crafting
+	screen_loc = ui_lay

```

Comment:
:eyes:

Reply:
Oh that explains why that part wasn't working right.